File: app/src/main/java/com/rtukpe/fixtures/data/model/Team.java
package com.rtukpe.fixtures.data.model;

import com.google.gson.Gson;
import com.google.gson.annotations.SerializedName;

public class Team {
    @SerializedName("id")
    public int id;
    @SerializedName("name")
    public String name;
    @SerializedName("shortName")
    public String shortName;
    @SerializedName("crestUrl")
    public String logo;

    @Override
    public String toString() {
        return new Gson().toJson(this);
    }
}

File: app/src/main/java/com/rtukpe/fixtures/data/model/Team.kt
package com.rtukpe.fixtures.data.model

import com.google.gson.Gson
import com.google.gson.annotations.SerializedName

class Team {
    @SerializedName("id")
    @JvmField
    var id: Int = 0
    @SerializedName("name")
    lateinit var name: String
    @SerializedName("shortName")
    lateinit var shortName: String
    @SerializedName("crestUrl")
    lateinit var logo: String

    override fun toString(): String {
        return Gson().toJson(this)
    }
}
